configuration-service/handlers/project.go

```diff
@@ -36,6 +36,13 @@ func PostProjectHandlerFunc(params project.PostProjectParams) middleware.Respond
 	common.LockProject(params.Project.ProjectName)
 	defer common.UnlockProject(params.Project.ProjectName)
 
+	rollbackFunc := func() {
```

Comment:
I would also add a test that verifies `rollbackFunc` is triggered in case of errors

Reply:
As discussed with @bacherfl, tests will be added in a follow up